### DISCUSSION TITLE
TCR/BCR clonotype analysis Seurat v5

### DISCUSSION THREAD

**Discussion by fificoder** [satijalab/seurat#8508] [General]
Hello,
Is there an analysis pipeline/code to analyze TCR/BCR (5' chemistry 10X) clonotype data in Seurat v5?
Thanks

**Comment by samuel-marsh** [comment]
Seurat does not directly have this functionality but some other packages can utilize Seurat object for such analysis.  Some suggestions of packages here:
https://www.10xgenomics.com/analysis-guides/immune-profiling-analysis-whats-next-after-cell-ranger

**Reply by fificoder** [reply]
Thank you!
